planemo/runnable.py

```diff
@@ -6,8 +6,8 @@
 import collections
 import os
 from distutils.dir_util import copy_tree
+from enum import auto, Enum
```

Comment:
@jmchilton `enum.auto` is available only on Python >=3.6 (in fact CircleCI builds are broken). Should we remove Python 3.5 support? It reached EOL on 2020-09-30.

Reply:
Yeah - go for it!

Reply:
Addressed in https://github.com/galaxyproject/planemo/pull/1099